rest-healthchek.go

```diff
@@ -0,0 +1,41 @@
+package sdk
+
+/*
+   Copyright 2016 Alexander I.Grafov <grafov@gmail.com>
+   Copyright 2016-2019 The Grafana SDK authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+	   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+   ॐ तारे तुत्तारे तुरे स्व
+*/
+
+import (
+	"fmt"
+)
+
+// Checks health of the grafana base url
+// Reflects GET BaseURL API call.
+func (r *Client) CheckHealth() error {
```

Comment:
The pattern in Go is typically to return `(bool, error)` in such cases because we want to differentiate between an error while trying to actually check this and the (non-)healthiness of Grafana. Actually, Grafana even returns such a response:
```
{
  "commit": "21bf8b71bc",
  "database": "ok",
  "version": "6.6.1"
}
```
So I'd suggest creating a new `struct` which has this information and then change the return type to `(HealthResponse, error)`. What do you think about this?